src/main/webapp/new-post.html

```diff
@@ -0,0 +1,97 @@
+limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>User Page</title>
+  <meta charset="UTF-8">
+
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+  <!-- Custom fonts for this template -->
+  <link href="https://fonts.googleapis.com/css?family=Raleway:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700,700i" rel="stylesheet">
+  <!-- Custom styles for this template -->
+  <link href="css/business-casual.min.css" rel="stylesheet">
+
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.css">
+  <link rel="stylesheet" href="/css/main.css">
+  <link rel="stylesheet" href="/css/map.css">
+  <script src="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBawpEnaMnt7vwsqGV_SaUEW7oppEDHJvg&libraries=places"></script>
+  <script src="/js/navigation-loader.js"></script>
+  <script src="/js/build-message-div.js"></script>
+  <script src="/js/new-post-loader.js"></script>
+  <script src="/js/map-loader.js"></script>
+
+</head>
+<body onload="initiateUI();">
+<!-- Website title header -->
+<h1 class="site-heading text-center text-white d-none d-lg-block">
+  <span class="site-heading-upper text-primary mb-3">Find Great Local Food</span>
+  <span class="site-heading-lower" id="page-title">Write A New Post</span>
+</h1>
+
+<!-- Navigation bar -->
+<nav class="navbar navbar-expand-lg navbar-dark py-lg-4" id="mainNav">
+  <div class="container">
+    <a class="navbar-brand text-uppercase text-expanded font-weight-bold d-lg-none" href="#">Website Name</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarResponsive">
+      <ul id="navigation" class="navbar-nav mx-auto">
+      </ul>
+    </div>
+  </div>
+</nav>
+
+<!-- Text on background -->
+<section class="page-section cta">
+  <div class="container">
+    <div class="row">
+      <div class="col-xl-9 mx-auto">
+        <!-- TODO: Format the form -->
+        <form id="message-form" action="/messages" method="POST">
+          Write your new post here:
+          <textarea name="text" id="message-input"></textarea>
+          <div class="mt-2 mb-1">
+            Type to search for location:
+            <input class="form-control z-depth-1 text-muted" type="text" name="location_input" value="eg. Maxwell Hawker Center" id="location_input">
```

Comment:
```suggestion
            <input class="form-control z-depth-1 text-muted" type="text" name="location_input" placeholder="eg. Maxwell Hawker Center" id="location_input">
```

May be worth checking out https://getbootstrap.com/docs/4.0/components/forms/#auto-sizing to see if we can put the input field and button on the same line because it looks quite messy now.